features.org

```diff
@@ -5,6 +5,9 @@
 
 #+begin_export html
 <style>
+  .justify {
+    text-align: justify;
+  }
```

Comment:
I'm a bit wary of justification because I know that on mobile it can easily produce 'rivers of whitespace'

Reply:
I'm missing something in my build which causes it not to match what you have running on your version of the site so I can't the styles and layout to match exactly, but the text more or less looks on mobile. If you are worried about it we could disable that style below a certain screen width.